options.c

```diff
@@ -1602,6 +1619,15 @@ void options_load(void)
 		if (errno != ENOENT)
 			error_msg("loading %s: %s", filename, strerror(errno));
 	}
+
+	/* check if charset supports the default format_clipped_text symbol */
+	if (strcmp(clipped_text_format, str_defaults[FMT_CLIPPED_TEXT].value) == 0) {
+		mbstate_t mbstate = {0};
+		char mb[MB_CUR_MAX];
+		if (wcrtomb(mb, L'…', &mbstate) == -1) {
+			clipped_text_internal = xstrdup("...");
+		}
+	}
```

Comment:
I haven't looked into the cause yet, but I noticed that left playlist pane and the file browser always seem to use `...` for some reason, even when the proper ellipsis is supported in other areas.

Reply:
Good catch. These places use format_str() from ui_curses.c which already adds "..." even in the current version.